From 79bc49f07261fe015351f9b5cfa26a403f8902cf Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Wed, 25 Apr 2018 16:34:40 +0200
Subject: [PATCH 0602/1051] arm64: dts: marvell: 7040-db: add comphy references
 to SATA ports

Adding phandle to the SATA port will allow AHCI driver to configure
appropriate marvell, comphy-cp110 accordingly.

Change-Id: I50b9fdb442ecf38cb3af8e668b3249954ca27ae0
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index eb64a5eb5535..0b165b767480 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -201,6 +201,12 @@
 
 &cp0_sata0 {
 	status = "okay";
+
+	sata-port@1 {
+		status = "okay";
+		/* Generic PHY, providing serdes lanes */
+		phys = <&cp0_comphy3 1>;
+	};
 };
 
 &cp0_usb3_0 {
-- 
2.17.1

